evap/staff/tools.py

```diff
@@ -375,6 +379,30 @@ def remove_user_from_represented_and_ccing_users(user, ignored_users=None, test_
     return remove_messages
 
 
+def remove_inactive_participations(user, test_run=False):
+    remove_messages = []
+    evaluations = []
+    if user.is_active and user.can_be_marked_inactive_by_manager:
+        evaluations = user.evaluations_participating_in.filter(
+            state=Evaluation.State.PUBLISHED,
+            vote_end_date__lt=datetime.today() - timedelta(days=settings.PARTICIPATION_DELETION_AFTER_INACTIVE_MONTHS),
+        )
+    if len(evaluations) > 0:
```

Comment:
In Python, programmers often try to go after this [notion of "Pythonicity"](https://stackoverflow.com/questions/25011078/what-does-pythonic-mean). Often, this boils down to ["why waste time say lot word when few word do trick"](https://www.youtube.com/watch?v=VvPaEsuz-tY). Here specifically, all well-behaved container-like structured in python allow to test for emptiness by simply "using as" boolean, i.e.:

```suggestion
    if evaluations:
```